src/components/Details.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useLocation, NavLink } from 'react-router-dom';
+import { BsArrowLeftCircle } from 'react-icons/bs';
+
+const Details = () => {
+  const location = useLocation();
+  const { state } = location;
+  return (
+    <section className="container details-card">
+      <NavLink
+        className="align-self-center arrow"
+        to="/"
+      >
+        <BsArrowLeftCircle className="details" />
+      </NavLink>
+      <h2 className="title">
+        {state.symbol}
+      </h2>
+      <div>
+        <span>
+          Volume:
+          {state.volume}
+        </span>
+        <span>
+          Ask price:
+          {state.askPrice}
+        </span>
+        <span>
+          Price change:
+          {state.priceChange}
+        </span>
+        <span>
+          Last price:
+          {state.lastPrice}
+        </span>
+        <span>
+          Count:
+          {state.count}
+        </span>
+        <span>
+          Bid price:
+          {state.bidPrice}
+        </span>
+        <span>
+          Open time:
+          {state.openTime}
+        </span>
+        <span>
+          Close time:
+          {state.closeTime}
+        </span>
+      </div>
+    </section>
+  );
```

Comment:
- [ ] I would recommend you stick to the design as much as possible. Just link the Homepage the details pages should have a banner section similar to the design.

  ![vechin-orig-detail](https://user-images.githubusercontent.com/36542986/189102737-769a4afa-3402-462c-a064-47d39f989984.jpg)

- [ ] Your details pages should also follow the composition and spacing between elements. In the design, the listed attributes are spaced out from the left to the right ends of the screen.

- [ ] Kindly make your font size and weights to be similar to the design as well.

- [ ] Kindly implement the stripe color pattern as we have in the design.

  Your Implementation | Given Design
  :---:|:---:
  ![ew](https://user-images.githubusercontent.com/36542986/189103629-a56b4c8e-d232-4643-81a7-3ec8e190d7bf.jpg) | ![debas-orig-detail](https://user-images.githubusercontent.com/36542986/189103632-1eaee0e1-172e-421b-b191-1a15bd97909c.jpg)